CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 2.1.x (Unreleased)
 
+## 2.1.1 (2022-11-14)
+
+- Bump thrift version to address https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-13949
+- Add support for lz4 compression
```

Comment:
as per semantic versioning shouldn't we do 2.2.0 instead?

Reply:
Sounds good to me! Thanks